frontend/packages/dev-console/src/components/pipelines/detail-page-tabs/pipeline-details/PipelineVisualizationTask.scss

```diff
@@ -2,7 +2,6 @@ $border-color: var(--pf-global--BorderColor--light-100);
 .odc-pipeline-vis-task {
   position: relative;
   width: 10em;
-  margin-top: 1.5em;
```

Comment:
nit: I understand this is an existing file... However, should we use pf-react-tokens instead later

Reply:
Not sure I understand. Your problem is with the sass styles or the hardcoded value? I believe I made the margin a pf variable when I moved it.

Reply:
this is not something urgent. While going through the code, I just thought that the remaining styles like width:10em, should they be converted to pf units in later PRs

Reply:
Ah gotcha... you mean the `width` haha... I thought you were talking about the `margin-top`

Reply:
This was all done at the time to make the nodes reactive to font size. The layout was dynamic too.
Now that we have a fixed layout and node sizes at can clean up the old code later

Reply:
On my cleanup list 😞